qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -353,6 +353,7 @@ def convert_play(self, shift, instruction):
         if isinstance(instruction.pulse, library.ParametricPulse):
             command_dict = {
                 'name': 'parametric_pulse',
+                'label': str(instruction.name or instruction.pulse.id),
```

Comment:
The name should always be based off the pulse name and not the name of the instruction.

Reply:
ok, I also updated the waveform qobj construction (it also took name from instruction).